src/NLog.Targets.MauiLog/NLog.Targets.MauiLog.csproj

```diff
@@ -1,72 +1,126 @@
-    <RepositoryType>git</RepositoryType>
-    <RepositoryUrl>https://github.com/NLog/NLog.Targets.MauiLog.git</RepositoryUrl>
-    <PackageTags>Maui;Xamarin;Forms;Android;MonoDroid;MacOS;iOS;LogCat;OSLog;NSLog;NLog;logging;log</PackageTags>
-    <PackageProjectUrl>https://github.com/NLog/NLog.Targets.MauiLog</PackageProjectUrl>
-    <PackageIcon>N.png</PackageIcon>
-    <PackageLicenseExpression>BSD-3-Clause</PackageLicenseExpression>
-    <PackageReadmeFile>README.md</PackageReadmeFile>
-    <PackageReleaseNotes>
-- Added support for UnhandledExceptionEventHandler in RegisterMauiLog
-- Updated to NLog v5.2.2
-- Added nuget-package README.md
-
-See https://github.com/NLog/NLog.Targets.MauiLog for documentation of NLog targets for MAUI
-    </PackageReleaseNotes>
-
-    <IsPackable>true</IsPackable>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <EnableTrimAnalyzer>true</EnableTrimAnalyzer>
-    <IsTrimmable>true</IsTrimmable>
-    <CheckEolWorkloads>false</CheckEolWorkloads>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <EmbeddedResource Include="ILLink.Descriptors.xml">
-      <LogicalName>ILLink.Descriptors.xml</LogicalName>
-    </EmbeddedResource>
-  </ItemGroup>
-
-  <ItemGroup>
-    <None Include="../../README.md" Pack="true" PackagePath="" />
-    <None Include="N.png" Pack="true" PackagePath="" Visible="false" />
-  </ItemGroup>
-  <Target Name="DownloadMissingContent" BeforeTargets="GenerateNuspec">
-    <DownloadFile SourceUrl="https://nlog-project.org/N.png" DestinationFolder="$(MSBuildThisFileDirectory)" />
-  </Target>
-
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'MonoAndroid90' Or '$(TargetFramework)' == 'net6.0-android' ">
-    <DefineConstants>$(DefineConstants);__ANDROID__</DefineConstants>
-    <SupportedOSPlatformVersion>21.0</SupportedOSPlatformVersion>
-  </PropertyGroup>
-
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'Xamarin.iOS10' Or '$(TargetFramework)' == 'Xamarin.Mac20' Or '$(TargetFramework)' == 'net6.0-ios' Or '$(TargetFramework)' == 'net6.0-macos' Or '$(TargetFramework)' == 'net6.0-maccatalyst' ">
-    <DefineConstants>$(DefineConstants);__APPLE__</DefineConstants>
-    <SupportedOSPlatformVersion>10.0</SupportedOSPlatformVersion>
-    <SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-macos'">10.14</SupportedOSPlatformVersion>
-  </PropertyGroup>
-
-  <ItemGroup Condition=" '$(TargetFramework)' == 'Xamarin.iOS10' Or '$(TargetFramework)' == 'Xamarin.Mac20' ">
-    <Reference Include="netstandard" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="NLog" Version="5.2.2" />
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
-  </ItemGroup>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFrameworks>net7.0-android</TargetFrameworks>
```

Comment:
Could this become:
```xml
    <TargetFrameworks Condition="'$(ContinuousIntegrationBuild)' != 'true'">netstandard2.0;net461</TargetFrameworks>
    <TargetFrameworks Condition="'$(ContinuousIntegrationBuild)' == 'true'">netstandard2.0;Xamarin.iOS10;Xamarin.Mac20;MonoAndroid90;net7.0-ios;net7.0-macos;net7.0-android;net7.0-maccatalyst</TargetFrameworks>
```

Reply:
I want the build-server to build nuget-package for all targets, but only build to netstandard2 and net461 on local machine.

Reply:
netstandard, xamarinios, xamarinmac are not maui targets. Is this supposed to be NLog.Targets.MauiLog or NLog.Targets.Xamarin?

Reply:
Not a MAUI-expert, but it seems that MAUI-Apps are able to consume libraries that doesn't have MAUI-specific-platforms (Ex. NetStandard). I would have thought one would only make platform-specific-targets when wanting different behavior than the default behavior (Ex. the one provided by NetStandard-target)